pkg/components/util/helm.go

```diff
@@ -15,10 +15,15 @@
 package util
 
 import (
+	"bufio"
```

Comment:
We shouldn't add more stuff to `util` package if we add new functions. I think they should go directly to `pkg/components` or `pkg/components/internal`.